wondrous-app/components/organization/pods/PodItem/index.tsx

```diff
@@ -0,0 +1,89 @@
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import RolePill from 'components/Common/RolePill';
+import PodIcon from 'components/Icons/podIcon';
+import MemberIcon from 'components/Icons/Sidebar/people.svg';
+import { LockedIconOutline, LockIconOutline } from 'components/Icons/userpass';
+import palette from 'theme/palette';
+import { useOrgBoard, useTokenGating } from 'utils/hooks';
+import { PodItemContainer, PodItemStatsContainer } from './styles';
+
+const PodItem = (props) => {
+  const { podData } = props;
+
+  const { userPermissionsContext } = useOrgBoard() || {};
+  const [tokenGatingConditions] = useTokenGating(podData?.orgId);
+
+  const podId = podData?.id;
+  const bgColor = podData?.color || palette.black85;
+  const podName = podData?.name;
+  const podDescription = podData?.description;
+  const contributorCount = podData?.contributorCount || 0;
+  const isPodTokenGated =
+    tokenGatingConditions?.getTokenGatingConditionsForOrg?.filter((condition) => condition.podId === podId).length > 0;
+  const role = userPermissionsContext?.podRoles[podId];
+
+  return (
+    <PodItemContainer>
+      <Grid sx={{ display: 'flex', alignItems: 'center', gap: '16px', maxWidth: '400px', overflow: 'hidden' }}>
```

Comment:
can we make a separate style component for this? The reason is the component's styles seems pretty big

Also MUI's Grid has props so you can do:

`<Grid display="flex" alignItems="center" gap="16"` />